lib/src/painters/player_wave_painter.dart

```diff
@@ -92,35 +92,31 @@ class PlayerWavePainter extends CustomPainter {
 
   void _drawWave(Size size, Canvas canvas) {
     final length = waveformData.length;
+    final halfWidth = size.width / 2;
+    final halfHeight = size.height / 2;
```

Comment:
Please prefer multiplication over division. You can use `* 0.5` here instead of `/ 2`.